src/reconstruct/helper.ts

```diff
@@ -129,6 +135,7 @@ export function getIndentString(indent: number): string {
 --helper function--
 */
 export function prettyPrintCodeToString(code: Code, lf = '\n'): string {
+	code = merge([code])
```

Comment:
wenn du für den code parameter in `merge` den spread operator benutzt, dann brauchst du hier kein extra Array :)

Reply:
Tatsächlich erspart das mir so evtl ein wenig arbeit bei den Tests, aber ich schau mal ob ich das noch änder